returnn/training.py

```diff
@@ -370,6 +371,28 @@ def create_returnn_config(
 
         return res
 
+    def check_blacklisted_parameters(self, returnn_config):
+        """
+        Check for parameters that should not be set in the config directly
+
+        :param ReturnnConfig returnn_config:
+        :return:
+        """
+        blacklisted_keys = [
+            "log_verbosity",
+            "device",
+            "num_epochs",
+            "save_interval",
+            "keep_epochs",
+        ]
+        for key in blacklisted_keys:
+            assert not (
+                key in returnn_config.config or key in returnn_config.post_config
```

Comment:
please use config.get() for this instead of manually checking config and post_config